src/app/main/main.c

```diff
@@ -1033,12 +1033,14 @@ sandbox_init_filter(void)
 
     OPEN_DATADIR("approved-routers");
     OPEN_DATADIR_SUFFIX("fingerprint", ".tmp");
+    OPEN_DATADIR_SUFFIX("ed25519_identity", ".tmp");
```

Comment:
Maybe this should be "ed25519-identity", to match "hashed-fingerprint" and "router-stability"?
(It appears we use underscores for keys, and dashes for user-readable text files.)

Let's see what @mikeperry-tor thinks.